app/components/AllOrders.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+export class AllOrders extends React.Component {
+  render () {
+    return (
+      <div>
+        <ul className="list-unstyled">
+          {
+            this.props.allOrders.map(order => {
```

Comment:
not 100% sure on this, but pretty darn sure: this component will be mounted and rendered *before* props are passed to it, but once the props are passed the component will be re-rendered. on the initial pass, props.allOrders will be undefined, which means that the `.map()` function will not run, which will cause an error, which will prevent our component from rendering.
the solution is to provide a check for the existence of props:
`props && props.allOrders.map()`